<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_computer</class>
 <widget class="QDialog" name="Edit_computer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:x-large; font-weight:600; color:#00ff00;&quot;&gt;Edit computer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_error_add_computer">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_error_computer_name">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="add_name">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_error_computer_type">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QRadioButton" name="electro_button">
       <property name="text">
        <string>Electronic</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mecha_button">
       <property name="text">
        <string>Mechatronic</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="trans_button">
       <property name="text">
        <string>Transistor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="other_button">
       <property name="text">
        <string>Other</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Year build:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_error_computer_build">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="add_build">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_edit_computer">
     <property name="text">
      <string>Edit computer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
